main.c

```diff
@@ -1177,10 +1181,18 @@ static void loadUsbDModule(void)
 //---------------------------------------------------------------------------
 static void loadUsbModules(void)
 {
+    int ret;
+
     loadUsbDModule();
-    if (have_usbd && !have_usb_mass && (USB_mass_loaded = loadExternalModule(setting->usbmass_file, &usb_mass_irx, size_usb_mass_irx))) {
+    if (have_usbd && !have_usb_mass && (USB_mass_loaded = loadExternalModule(setting->usbmass_file, NULL, 0))) {
         delay(3);
         have_usb_mass = 1;
```

Comment:
This can probably be removed?
What did the delay(3) do before this PR?

Reply:
I guess it delayed by some arbitrary amount just in case devices take longer to show up. But I don't think it's important here because USB modules are loaded after config is loaded